tests/syrupy/extensions/amber/test_amber_serializer.py

```diff
@@ -233,3 +233,22 @@ def test_ordered_dict(snapshot):
 def test_many_sorted(snapshot):
     for i in range(25):
         assert i == snapshot
+
+
+def function_to_test(
+    var1, var2="test_val", var3: str = "test_val2", *, kwvar1, kwvar2="some_val"
+) -> str:
+    return "2"
+
+
+def test_function_in_file(snapshot):
+    assert snapshot() == function_to_test
```

Comment:
```suggestion
    assert snapshot == function_to_test
```

no need to invoke snapshot